front/app/containers/Admin/projects/project/general/components/GeographicAreaInputs.tsx

```diff
@@ -187,7 +187,6 @@ const GeographicAreaInputs = ({
             options={areaOptions}
             value={selectedAreaValues}
             onChange={handleAreaSelectionChange}
-            disabled={areaType !== 'selection'}
```

Comment:
Doesn't have any effect because component is only rendered when areaType is 'selection'.